### PR TITLE
Add option to espanol/torre/huida

### PR DESCRIPTION
**Pull request by albsama**
Please review this, @casey-collab

### CODE REVIEW

File: espanol/torre/huida/huida.md [hunk 1]
```diff
@@ -1 +1,3 @@
 No se puede medir la fuerza del enemigo y su estimacion supone una fuerza muy alta.
+
+Por lo que el consejero decidió de nuevo [subir a la torre](torre/torre.md)
```
Comment (casey-collab) [inline espanol/torre/huida/huida.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: espanol/torre/huida/huida.md (hunk 1 above)
Comment (albsama) [inline espanol/torre/huida/huida.md]: Broken link fixed, @casey-collab

**Comment by albsama** [comment]
Done. Broken link fixed, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :fire:

**Pull request merged**